recipe/meta.yaml

```diff
@@ -1,41 +1,42 @@
 {% set name = "cellxgene-census" %}
 {% set last_name = "cellxgene_census" %}
-{% set version = "1.15.0" %}
+{% set version = "1.16.2" %}
 # shasum -a 256 of https://pypi.io/packages/source/c/cellxgene-census/cellxgene_census-i.j.k.tar.gz
 # See also
 # https://pypi.org/project/cellxgene-census/#files
-{% set sha256 = "db059d2e615ba7d2a7f59db8f7582dae7a0adccac1449d859af6bec2e040d072" %}
+{% set sha256 = "c007031b1e42b320a191ff1f20ea4099ea935fe5416dffcd710b0e4a94c28ce1" %}
+# https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python
+# Bump minimum in conda_build_config.yaml and rerender
+{% set python_min = python_min|default("0.1a0") %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ last_name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ last_name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    - remove-tiledbsoma-upper-bound.patch
```

Comment:
The patch I created for 1.15.0 is no longer needed because there is no longer an upper bound on tiledbsoma-py